<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SanPham</class>
 <widget class="QDialog" name="SanPham">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tên sản phẩm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_SanPham">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>570</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_TenSanPham">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Loại sản phẩm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_LoaiSanPham">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_SoLuong">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Giá bán</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_GiaBan">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_SoLuong">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Số lượng</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID sản phẩm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ID_SanPham">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Them">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Thêm </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>261</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Giá nhập</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_GiaNhap">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Sua">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Sửa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Xoa">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Xóa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Xem">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Xem</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>419</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Tìm kiếm</string>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tìm</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView_SanPham">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>741</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>170</y>
     <width>392</width>
     <height>236</height>
    </rect>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>label</zorder>
  <zorder>label_SanPham</zorder>
  <zorder>lineEdit_TenSanPham</zorder>
  <zorder>label_2</zorder>
  <zorder>lineEdit_LoaiSanPham</zorder>
  <zorder>lineEdit_SoLuong</zorder>
  <zorder>label_3</zorder>
  <zorder>lineEdit_GiaBan</zorder>
  <zorder>label_SoLuong</zorder>
  <zorder>label_5</zorder>
  <zorder>lineEdit_ID_SanPham</zorder>
  <zorder>pushButton_Them</zorder>
  <zorder>pushButton_Sua</zorder>
  <zorder>pushButton_Xoa</zorder>
  <zorder>pushButton_Xem</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>tableView_SanPham</zorder>
  <zorder>calendarWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
